src/consensus/vote.cpp

```diff
@@ -543,51 +543,48 @@ bool VoteManager::voteValidation(std::shared_ptr<Vote>& vote, size_t dpos_total_
 bool VoteManager::pbftBlockHasEnoughValidCertVotes(SyncBlock& pbft_block_and_votes, size_t dpos_total_votes_count,
                                                    size_t sortition_threshold, size_t pbft_2t_plus_1) const {
   if (pbft_block_and_votes.cert_votes.empty()) {
-    LOG(log_er_) << "No any cert votes! The synced PBFT block comes from a "
-                    "malicious player.";
+    LOG(log_er_) << "No any cert votes! The synced PBFT block comes from a malicious player.";
+    return false;
+  }
+  if (pbft_block_and_votes.cert_votes.size() != pbft_2t_plus_1) {
```

Comment:
so if we have more votes it is not ok?

Reply:
@gaoqi8649  the log is inaccurate because you say doesnt have enough cert votes but it could in fact have too many.

@MatusKysel thats a good question.   Sending excessive cert votes while syncing i think is indeed a bad behavior.   Seems not worth it to verify them in hopes we just got too many....

Reply:
@MatusKysel Since for good nodes, they will only save 2t+1 cert votes in DB. So for any good nodes, that's not possible sending out over 2t+1 cert votes.

Reply:
@justinsnapp fixed log error, thanks